### ISSUE TITLE
Create research readout or documentation (for dashboard design and other direction setting research)

### ISSUE THREAD

**Issue by jenniferthibault** [18F/openFEC-web-app#1157]
We should be capturing what we're learning about the dashboard design as it's in progress:
- research approach
- what we've tested and what we've learned
- overall conclusions

We have various synthesis documents, but this feature is so big and so large that we (and future us) will benefit a lot from a clearly organized, external-facing research readout. 

cc @noahmanger @emileighoutlaw @brethauer

**Comment by emileighoutlaw** [comment]
This is a great idea, and I'm so glad you brought it up, Jen. 

I'd love to know what (if any) kind of templates projects use to capture and store usability results for future reference. @brethauer, do you have any experience putting together documentation like this?

**Comment by brethauer** [comment]
Yes! This has been on my to-do list (to aggregate all of this last couple week's worth of research/synthesis into one place.)  Thank you for the nudge! :+1:

**Comment by brethauer** [comment]
@jenniferthibault see: https://github.com/18F/openFEC/issues/1596 and https://docs.google.com/document/d/1Pt-0TtdxS1RpKj_s-0vdAWFwMRLtaQeqG2MO-6B9DqA/edit#

**Comment by noahmanger** [comment]
Excellent. Thank you @brethauer . Closing this.

**Comment by jenniferthibault** [comment]
@noahmanger sorry I don't think this was ready to close yet. Reopening for discussion & refinements.

@brethauer @emileighoutlaw I read through and left comments and feedback in the doc. I love having all the things we've learned and done in one spot! My comments probably seem like a lot...but I'm trying to approach this with the mindset of someone who didn't sit through many of the interviews, and is looking for background and rationale for what exists. If that's off-base, or if you'd rather receive feedback on this in another way, please let me know!

Feedback broadly fall into a few categories:
- Clarifying points so that we don't leave future readers guessing what we mean
- Questions where we could come off as over-attributing what we heard
- Making a complete record of what we tested (through visuals or other links)

And a few questions overall:
- Where do you think this should live? 
- What format do you think it should live in?
- Is this a template and format you'd envision we replicate for future research documentation for the FEC?

I know of a few examples of other folks on other projects who have done something like this, but there are undoubtedly more. 
- For USCIS, Brad Nunnally and his team put together a report of their research documentation to give to the contractor and partner agency who is taking over the design. It's more formal than we probably need, but we can likely learn from the template. It's in a private repo, but I got the markdown files and can share with the slack channel.
- For MyUSA, Liz Goodman also set up some templates where she'd record synthesis and research findings, and she used [a GitHub repository ](https://github.com/18F/myusa-ux). I find the flow of documents like [this](https://github.com/18F/myusa-ux/blob/master/research/concept-evaluation-201501.md) and [this](https://github.com/18F/myusa-ux/blob/master/research/front%20page%2020150222.md) that use specific screenshots supported by quotes really helpful. I know she has one doc somewhere which she calls her holy grail. I'll ask her when she's online!

**Comment by noahmanger** [comment]
Oops! Sorry for jumping the gun on this. Really good, thoughtful points @jenniferthibault . Really helpful to have your perspective on it, and agree throughout that further clarification will be good to document (especially with screenshots).

Only thing I'm wondering is to what extent do we want this to be a living document? As we talked about at the beginning of this work, this is a continuous discovery process. So while we did discovery and prototype testing, we're going to continue to learn a lot as we release functionality. Do we want to "put a pin" in this document as the conclusion of this stage? Or structure it in a way that will accommodate future learnings (vs. capturing those in separate places)?
